docs/user/syncing-apprepository-webhook.md

```diff
@@ -0,0 +1,159 @@
+
+### Creating the Kubernetes ServiceAccount, ClusterRole and RoleBinding objects
+
+First, the `ServiceAccount` and the `ClusterRole` objects are created. In this example, the ServiceAccount *apprepositories-webhook* is being created.
+
+```bash
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: apprepositories-webhook
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: apprepositories-webhook
+rules:
+  - apiGroups:
+      - kubeapps.com
+    resources:
+      - apprepositories
+    verbs:
+      - get
+      - update
+EOF
+```
+
+Next, the `ClusterRole` *apprepositories-webhook* (with `get` and `update` permissions over `apprepositories`) has to be associated to a certain (or multiple) namespace(s) by means of a `RoleBinding`:
+
+> To bind it in multiple namespaces, create as RoleBinding (changing `namespace: my-namespace` to match your needs) as you need.
+
+```bash
+cat <<EOF | kubectl apply -f -
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: apprepositories-webhook
+  namespace: my-namespace
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: apprepositories-webhook
+subjects:
+- kind: ServiceAccount
+  name: apprepositories-webhook
+  namespace: default
+EOF
+```
+
+### Retrieving the ServiceAccount token
+
+The `<TOEKN>`  for the ServiceAccount *apprepositories-webhook* is retrieved:
+
+```bash
+kubectl get secret $(kubectl get serviceaccount apprepositories-webhook -o jsonpath='{range .secrets[*]}{.name}{"\n"}{end}' | grep apprepositories-webhook) -o jsonpath='{.data.token}' -o go-template='{{.data.token | base64decode}}' && echo
+```
+This value will be the Bearer token to be passed in the `Authentication` HTTP header.
+
+## Configuring a webhook in Harbor
+
+A high-level description of the main steps is presented herein; please refer to the [official Harbor documentation](https://goharbor.io/docs/1.10/working-with-projects/project-configuration/configure-webhooks/) for further information.
+
+1. Log in to the Harbor interface with an account that has at least project administrator privileges.
+2. Go to  *Projects*, select a project, and select  *Webhooks*.
+3. Enter this URL: `https://<KUBEAPPS_URL>/api/v1/clusters/<CLUSTER_NAME>/namespaces/<NAMESPACE_NAME>/apprepositories/<APPREPO_NAME>/refresh`. Modify `<KUBEAPPS_URL>` , `<CLUSTER_NAME>`, `<NAMESPACE_NAME>`, `<APPREPO_NAME>` to match your needs. 
+	> For instance: `https://<KUBEAPPS_URL>/api/v1/clusters/default/namespaces/kubeapps/apprepositories/bitnami/refresh` will update the App Repository `bitnami` in the namespace `kubeapps` of the cluster `default`.
+	
+4.  As authentication header enter `Bearer <TOKEN>`
+	> Note `<TOKEN>`is the value retrieved [before](#retrieving-the-serviceaccount-token).
+	
+5. Select as many events as you need to trigger notifications.
```

Comment:
Perhaps you could append to step 5: " The image below shows the common events that you would want for triggering a resync." and then updating the image to just select the relevant ones? (I could be missing something, but I think it'd just be "Chart Uploaded" and "Chart Deleted"? (possibly also Artifact pushed / deleted for future OCI stuff?)